modules/ossm-exposing-service-using-istio-gateway-and-virtualservice.adoc

```diff
@@ -0,0 +1,133 @@
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-exposing-service-using-istio-gateway-and-virtualservice_{context}"]
+= Exposing a service using an Istio gateway and a VirtualService
```

Comment:
I think we called the procedure something like this in the content architecture call yesterday.

```suggestion
= Exposing a service using the Istio Gateway and VirtualService resources
```

Reply:
done